packages/web3-common/src/eth_execution_api.ts

```diff
@@ -0,0 +1,298 @@
+import {
+	Address,
+	HexString,
+	HexString256Bytes,
+	HexString8Bytes,
+	HexString32Bytes,
+	HexStringBytes,
+	HexStringSingleByte,
+	Uint,
+	Uint256,
+} from 'web3-utils';
+
+// The types are generated manually by referring to following doc
+// https://github.com/ethereum/execution-apis
+
+export interface AccessListEntry {
+	readonly address?: Address;
+	readonly storageKeys?: HexString32Bytes[];
+}
+export type AccessList = AccessListEntry[];
+export type TransactionHash = HexString;
+export type Uncles = HexString32Bytes[];
+export type BlockTag = 'earliest' | 'latest' | 'pending';
+export type BlockNumberOrTag = Uint | BlockTag;
+
+export interface TransactionCall {
+	readonly from?: Address;
+	readonly to: Address;
+	readonly gas?: Uint;
+	readonly gasPrice?: Uint;
+	readonly value?: Uint;
+	readonly data?: HexString;
+}
+
+export interface BaseTransaction {
+	readonly to?: Address | null;
+	readonly type: HexStringSingleByte;
+	readonly nonce: Uint;
+	readonly gas: Uint;
+	readonly value: Uint;
+	readonly input: HexStringBytes;
+}
+
+export interface Transaction1559Unsigned extends BaseTransaction {
+	readonly maxFeePerGas: Uint;
+	readonly maxPriorityFeePerGas: Uint;
+	readonly accessList: AccessList;
```

Comment:
chainId is missing: `"chainId": {
				"title": "chainId",
				"description": "Chain ID that this transaction is valid on.",
				"$ref": "#/components/schemas/uint"
			}`